cluster/manifests/skipper/skipper-redis.yaml

```diff
@@ -0,0 +1,45 @@
+{{ if eq .ConfigItems.skipper_clusterratelimit "true"}}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    application: skipper-ingress-redis
+    version: v4.0.9
+  name: skipper-ingress-redis
+  namespace: kube-system
+spec:
+  replicas: 2
```

Comment:
I suggest to always create the statefulset (and the service), and set the replicas to either 2 or 0 based on `skipper_clusterratelimit`. Otherwise you would have to manually cleanup after disabling it.

Reply:
I think this feature should go GA and it's not as easy to jsut change the number of replicas, but you need to modify the skipper-ingress params, too.
So we would need to build the string we pass to skipper with the same arg.
Feel free to contribute to this PR, otherwise I would like ignore it for now and enhance it later.

Reply:
I'm not talking about making the number of replicas configurable. What I meant was moving the `if` to `replicas`:

    replicas: {{ if eq .ConfigItems.skipper_clusterratelimit "true"}}2{{else}}0{{end}}

Which allows us to enable or disable the flag by changing just one config item and then waiting. In the PR's version, disabling it means we have to manually login to the cluster and delete the statefulset as well.

Reply:
done